src/Utils/TreeUtils.test.ts

```diff
@@ -0,0 +1,32 @@
+
+        test('inTree', () => {
```

Comment:
Strings in `test` are supposed to explain the test. `inTree` is not appropriate. E.g. "given tree and dialog within tree should return true"